templates/launch.html

```diff
@@ -460,6 +463,57 @@ <h3>Runner Data</h3>
         document.getElementById('schema-url-survey-type').selectedIndex = -1;
     }
 
+    function retrieveSchemaName() {
+        let schemaName = localStorage.getItem("schema_name");
+        if (schemaName) {
```

Comment:
Since its a dev tool, doesn't really matter too much, but you can avoid getting a console error when the stored schema is no longer in the list if you check the schema is in the list first
```
const availableSchemas = [...document.querySelector("#schema_name").options].map(x => x.value)
if (schemaName && availableSchemas.includes(schemaName)) {
```

Reply:
Added, thanks!